UNITY.md

```diff
@@ -164,4 +164,103 @@ Note that even though the `<PlacementRoot>` is identical between projects each w
 - `<LookAt>` is calculated by generating a direction vector (`<Position> - target`)
   - Note that `target` must be converted to world space before calculating the direction vector
   - Unity has a built in `LookRotation()` function that takes the direction vector and `up`
-- `Normal` is calculated by **(TODO #63)**
+- `Normal` is calculated by **TODO: 63**
+
+## ObjectRoot
+
+Each `<Object>...` inside `<ObjectRoot>` corresponds to a single GameObject in Unity. The `<Placement>` is set using `Placement.SetTransform`, see [above](#placement).
+
+### Object
+
+- `name`: gameObject.name
+- `<Color>`: gameObject.[content].color
+  - Exactly how the color is applied
+- `<Visible>`: gameObject.active
+- `<Lighting>`: TODO (76)
+- `<ClickThrough>`: TODO (76)
+- `<AroundSelfAxis>`: TODO (76)
+- `<Scale>`: gameObject.localScale
+  - Sent as a variable to `Placement.SetTransform()`
+- `<Content>`: An `<xs:choice>` between several content types, [see below](#content)
+- `<LinkRoot>`: Adds a `Button` parent to `<Content>` in Unity with a list of defined actions (optional)
+  - A "link" in the original project acts the same as a `Button` in Unity
+
+### Content
+
+`<Content>` always contains exactly one child, a choice between several different types of content. Each type corresponds (roughly) to a different GameObject type in Unity and is translated as such. Note that `<Content>` does NOT create a child object in Unity, it is attached to the GameObject created by `<Object>` directly.
+
+#### None
+
+`<Object>` is an empty GameObject. This is the default cause in the switch statement (`CLI.TranslateGameObjects`).
+
+#### Text
+
+Text in Unity is defined by `TextMeshPro`. Some properties are set for all `TextMeshPro` objects to closer match the text defined in the original project (font size, word wrapping and overflow settings, etc.)
+
+- `<text>`: The string physically displayed on screen
+- `horiz-align`: The horizontal alignment of `<text>` (TMPro.HorizontalAlignmentOptions)
+- `vert-align`: The vertical alignment of `<text>` (TMPro.VerticalAlignmentOptions)
+- `font`: A path to a font asset
+  - Several "default" font materials have already been created in `Assets/Resources/Materials/Fonts`
+  - `font` is a .ttf file, if the material hasn't been created the script attempts to create it
+  - LiberationSans SDF (The default TextMeshPro font) is used as a fallback
+- `depth`: The size of the text along the z axis
+
+The vertex and material color (`.color` and `.faceColor`) of the TextMeshPro object are set using the parent `<Color>` tag.
+
+#### Image
+
+TODO 65
+
+- `filename`: Path to the image file
+
+#### StereoImage
+
+TODO 66
+
+- `left-image`: Path to the image file on the left side
+- `right-image`: Path to the image file on the right side
+
+#### Model
+
+TODO 67
+
+- `filename`: Path to the 3d object file
+- `check-collisions`: Whether or not other objects can pass through
+  - A Unity mesh is added over the object
+
+#### Light
+
+TODO 68
```

Comment:
Actually probably a good idea for these big ones because I've been updating the issue as I find info about how Unity handles the different object types